.maint/paper_author_list.py

```diff
@@ -46,23 +46,26 @@ def _aslist(inlist):
     ]
 
     print(
-        'Some people made commits, but are missing in .maint/ '
-        'files: %s.' % ', '.join(unmatched),
+        'Some people made commits, but are missing in .maint/ ' 'files: {}.'.format(
```

Comment:
Isn't there a rule to catch these joined strings?


```suggestion
        'Some people made commits, but are missing in .maint/ files: {}.'.format(
```

Reply:
:smile: That was an issue with black, and it looks like ruff isn't any better.